end-to-end-test/mocks/auth/landing_company.ts

```diff
@@ -0,0 +1,1841 @@
+import { Context } from '../../utils/mocks/mocks';
+
+export default function mock_landing_company(context: Context) {
+    if ('landing_company' in context.request && context.request.landing_company === 'th') {
```

Comment:
We have ts error here :thinking: 

![image](https://github.com/binary-com/deriv-app/assets/93518187/ab018dcb-dff9-479f-9947-2fdfbbdb2a89)